src/lib/media/style.css

```
@@ -36,3 +36,115 @@ body {
+
+/* Editing state - when input is visible */
+.config-input.editing {
+  display: block !important;
+}
+
+/* Ensure display values don't look like inputs */
+.config-value {
+  background: transparent;
+  border: none;
+  padding: 2px 4px;
+  word-wrap: break-word;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)